### PR TITLE
Fix failing tests

### PR DESCRIPTION
**Pull request by superkhau**
- Fix missing package.json git repo info
- Assign missing baseUrl in server/server.js
- Add smoke test
- Allow mocha globals in linter
- Add mocha to dev deps

### CODE REVIEW

**Comment by superkhau** [comment]
@slnode test please

**Comment by superkhau** [comment]
@slnode test please

**Comment by superkhau** [comment]
@slnode test please

**Comment by superkhau** [comment]
@bajtos Merging without review as this is a simple fix for CI.

File: package.json [hunk 1]
```diff
@@ -3,7 +3,9 @@
   "version": "1.0.0",
   "main": "server/server.js",
   "scripts": {
-    "pretest": "jshint ."
+    "pretest": "jshint .",
+    "start": "node server/server",
```
Comment (bajtos) [inline package.json]: Nitpick: `node .` would be better, there is no need to duplicate `server/server` - see L4 `"main": "server/server.js"`

Comment (superkhau) [inline package.json]: Fixed.

**Pull request merged**